zanata-war/src/main/java/org/zanata/service/impl/ConfigurationServiceImpl.java

```diff
@@ -105,7 +105,6 @@ public String getConfig() {
                             .append("\n")
                             .append(makeProjectTypeSection(getProjectType()))
                             .append("\n");
-            appendLocalesIfPresent();
```

Comment:
@seanf all generated zanata.xml files will no longer have a `<locales>` section. That includes the zanata.xml in zip downloads. It should not change anything about po files.